ui/widgets/nuxeo-selectivity.js

```diff
@@ -6962,22 +6957,18 @@ input[type='text'].selectivity-multiple-input:focus {
 
 .selectivity-single-select {
   cursor: pointer;
-  min-height: 2em;
-  padding: 5px;
+  padding: 5px 5px 0px 5px;
   position: relative;
   -webkit-box-sizing: content-box;
   box-sizing: content-box;
 }
 
 .selectivity-single-select-input {
   opacity: 0;
+  display: none;
```

Comment:
Do we really need this? I'm not sure about its purpose. Could you please provide further details?